packages/bcer-data-portal/app/src/constants/validate.ts

```diff
@@ -20,4 +20,12 @@ export const notificationValidationSchema = yup.object({
 export const faqValidationSchema = yup.object({
   title: yup.string().required('Question cannot be empty'),
   description: yup.string().required('Response cannot be empty')
-})
\ No newline at end of file
+})
+
+export const locationInformationValidationSchema = yup.object({
```

Comment:
Can you use existing validation if they exist? 

Or are they only on the retailer side?

Reply:
yes, they are only on the retailer side